tests/api/Jalasoft.TeamUp.Resumes.API.Tests/GetResumesTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Jalasoft.TeamUp.Resumes.API.Tests
+{
+    using Jalasoft.TeamUp.Resumes.API.Controllers;
+    using Jalasoft.TeamUp.Resumes.Core.Interfaces;
+    using Jalasoft.TeamUp.Resumes.Models;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Moq;
+    using Xunit;
+
+    public class GetResumesTests
+    {
```

Comment:
In this case, the name could be GetResumesApiTests

Reply:
similar answer to above's one